examples/tabs/js/alt.js

```diff
@@ -0,0 +1,4 @@
+var Alt = require('../../../src/alt')
+var alt = new Alt()
+
+module.exports = alt
```

Comment:
If you look at the alt source, there are also no semicolons (the linter is even set to not accept them). JavaScript's automatic semicolon insertion works pretty well, so semicolons are mostly preferential (though there are a few edge cases where this can bite you if you are not careful).  I include them in most of my projects out of habit, but I think that it is more important to be consistent with the rest of the repo as it leads to less confusion in the end.